lib/config.js

```diff
@@ -40,9 +40,14 @@ config.mongodb.authentication = {
   // used by MongoDB < 3.0
   // authMechanism: 'MONGODB-CR'
 };
-config.mongodb.connectOptions = {
+
+const writeConcern = config.mongodb.writeConcern = {
   w: 'majority',
-  j: true,
+  j: true
```

Comment:
I'm a fan of the dangling comma, so I would prefer not seeing them removed.  They are legal based on current linting rules so I don't see any reason it would need to be removed.

Reply:
This is a change from previous linting rules. When I joined DB a previous PR was held up do to a dangling comma. I have no issue with them either so I will re-add it. We might want to make a lint rule for it: https://eslint.org/docs/rules/comma-dangle

I added an issue here:
https://github.com/digitalbazaar/eslint-config-digitalbazaar/issues/49